### ISSUE TITLE
Dimensionality Issue with scipy.solve_ivp

### ISSUE THREAD

**Issue by huang-liang1996** [COVIDAnalytics/DELPHI#19]
Hi Covid Analytics Team,

Thank you so much for providing the source code for the model. I have some concerns with the code, especially the scipy.solve_ivp in the policy prediction process. I kept getting this error:
```
\Anaconda3\lib\site-packages\scipy\integrate\_ode.py:1348: UserWarning: lsoda: Excess accuracy requested (tolerances too small).
  self.messages.get(istate, unexpected_istate_msg)))
Results:  -1 Unexpected istate in LSODA.
```
and measures except "No measure", "Restrict Mass Gathering" and "Lockdown" are skipped. When I modify the tolerance to larger numbers, the results get negative and counter-intuitive.

I changed the method in solve_ivp to LSODA since it generated the least errors and tried to print why the process is not completed (as shown in the "Results" line).

I have also tried to run the prediction parameters through Mathematica while it does not work for the solve_ivp as well. Could I know how I could make improvements on the model to include more policies, just as what is shown on the website?

Thank you very much!

**Comment by MichaelLLi** [comment]
could you provide more details on what file you are using and how you are using it? It would be great if you can provide a MWE.  LSODA should generally not be utilized as a method for these differential equations due to its numerical behavior.

**Comment by huang-liang1996** [comment]
> could you provide more details on what file you are using and how you are using it? It would be great if you can provide a MWE. LSODA should generally not be utilized as a method for these differential equations due to its numerical behavior.

Hi I am currently using these files DELPHI_policy_predictions.py and DELPHI_model.py. 

I applied on California and Illinois data. I removed other countries' and provinces' data set and policy data so only California and Illinois past data for disease spread and policies are left. I modify the dates input inside so I can output the parameters from 27 May until 31 May through DELPHI_model.py. Then I adopted the DELPHI_policy_predictions.py and produce the errors as mentioned above. 

Thank you!
